Makefile

```diff
@@ -122,9 +122,9 @@ CUOBJ = $(patsubst %.cu, build/%_gpu.o, $(CUSRC))
 
 # extra operators
 ifneq ($(EXTRA_OPERATORS),)
-	EXTRA_SRC = $(wildcard $(patsubst %, %/*.cc %/*/*.cc, $(EXTRA_OPERATORS)))
+	EXTRA_SRC = $(wildcard $(foreach DIR, $(EXTRA_OPERATORS), $(foreach PAT, *.cc */*.cc, $(DIR)/$(PAT))))
```

Comment:
are you sure? From doc of make:
For example,

$(patsubst %.c,%.o,x.c.c bar.c)
produces the value ‘x.c.o bar.o’.